curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/nest-an-anchor-element-within-a-paragraph.md

```diff
@@ -32,18 +32,18 @@ Next is the *anchor* element `<a>` (which requires a closing tag `</a>`):
 `target` is an anchor tag attribute that specifies where to open the link. The value `_blank` specifies to open the link in a new tab. The `href` is an anchor tag attribute that contains the URL address of the link:  
 
 ```html
-<a href="http://freecodecamp.org"> ... </a>
+<a href="http://freecodecamp.org" target="_blank"> ... </a>
 ```
 
 The text, `link to freecodecamp.org`, within the `a` element is called <dfn>anchor text</dfn>, and will display the link to click:
 
 ```html
-<a href=" ... ">link to freecodecamp.org</a>
+<a href=" ... " target="_blank">link to freecodecamp.org</a>
 ```
 
 The final output of the example will look like this:  
 
-Here's a [link to freecodecamp.org](http://freecodecamp.org) for you to follow.
+Here's a <a href="http://freecodecamp.org" target="_blank">link to freecodecamp.org</a> for you to follow.
```

Comment:
Here as well

```suggestion
Here's a <a href="https:/www./freecodecamp.org" target="_blank">link to freecodecamp.org</a> for you to follow.
```

Reply:
@nhcarrigan Do you think we need `rel="noopener noreferrer"` too?

Reply:
You have a typo in the suggestion here 😉 

I think those aren't needed from a *security* standpoint here, because we're linking to ourselves. However, I do agree that covering them wouldn't hurt.

That being said, I think that's outside of the scope of this PR - we should create an issue to update the examples and instruction text, possibly.

Reply:
Oops 😝 

Yeah I agree that this shouldn't block the PR (that's why I didn't request the change). I also think it's easier to maintain the consistency if we audit and make the update in the same PR, rather than randomly.

I'll let you sign off on the PR then 🙂